src/index.rst

```diff
@@ -91,6 +91,146 @@ The first parameter (computing_cloud) defines the PanDA cloud associated
+    So for ``push`` mode, one pilot is bound with one user job. In the mode, one pilot
+    can run only one job. Since the pilot is submitted after
+    the user job is created, pilot will be submitted with exact requestMemory of the job.
+    For example, if a job requests 20GB memory. The job will be scheduled to
+    ``SLAC_Rubin_Extra_Himem``. If this queue was ``pull`` mode, the pilot would be submitted
+    with 220GB (the maxRSS). However, since this queue is ``push`` mode, the pilot will be
+    submitted with the requestMemory 20GB.
+
+  * The whole idea is that we think there will be a lot of jobs with not much memory. So
+    we use ``pull`` mode to make it efficient. We think the number of jobs which request
+    extra high memory is not big. But for these jobs, the range of requestMemory can be big,
+    so we use ``push`` mode to efficiently use the memory.
+
+There is another special queue *SLAC_Rubin_Merge*, its memory range is from 0GB to
+500GB (The maximum memory one machine at SLAC can provide). However, this queue is
+brokeroff. It means PanDA will not schedule andy jobs to it. It can only accept jobs when
+users specify the queue name.
+
+*SLAC_TEST* currently is still on production. It will be moved for development only in
+the future.
+
+
+.. list-table:: USDF (SLAC) PanDA Queues
+   :widths: 50 25 25 25 25 25
+   :header-rows: 1
+
+   * - PanDA Queue
+     - slurm queue
+     - minRSS
+     - maxRSS
+     - Harvester mode
+     - Brokerage
+   * - SLAC_Rubin
+     - rubin
+     - 0GB
+     - 4GB
+     - pull
+     - on
+   * - SLAC_Rubin_Medium
+     - rubin
+     - 4GB
+     - 8GB
+     - pull
+     - on
+   * - SLAC_Rubin_Himem
+     - rubin_himem
+     - 8GB
+     - 18GB
+     - pull
+     - on
+   * - SLAC_Rubin_Extra_Himem
+     - rubin_extra_himem
+     - 18GB
+     - 220GB
+     - push
+     - on
+   * - SLAC_Rubin_merge
+     - rubin_merge
+     - 0GB
+     - 500GB
+     - push
+     - off
+   * - *SLAC_Test*
+     - rubin
+     - 0GB
+     - 4GB
+     - pull
+     - on
+
+How to submit jobs to USDF
+--------------------------
+
+
+  * Only request memory and let PanDA do the scheduling(do not define *queue*).
+    Here is an example::
+
+       computeCloud: "US"
+       computeSite: "SLAC"
+       requestMemory: 2048
+
+       pipetask:
+           pipetaskInit:
+               requestMemory: 4000
+
+       executionButler:
+           requestMemory: 4000
+
+  * Another example by specifying queues (Here a *queue* is defined)::
+
+       computeCloud: "US"
+       computeSite: "SLAC"
+       requestMemory: 2048
+
+       pipetask:
+           pipetaskInit:
+               requestMemory: 4000
+
+       executionButler:
+           # *requestMemory is still required here.*
+           # *Otherwise it can be schedule to the merge*
+           # *queue, but the requestMemory is still 2048*
+           requestMemory: 4000
+           queue: "SLAC_Rubin_Merge"
```

Comment:
I would ask @hsinfang about this example.  While this is correct, I think the plans are to have this queue specified in a SLAC-specific yaml file that is included in the user's run-specific submit yaml file.

Reply:
I agree that we'd want this configuration in the USDF's default yaml @zhaoyuyoung is working on.  

Indeed the merge job is a bit special. 
Do I understand correctly this matters for the push mode only? If so, SLAC_Rubin_Extra_Himem can be the example too?

Reply:
Would this be a correct example? 
```
 payload:
     forcedPhotCoadd:
         # *requestMemory is still required here.*
         # *Otherwise it can be schedule to the extra himem*
         # *queue, but the requestMemory is still 2048*
         requestMemory: 20000
         queue: "SLAC_Rubin_Extra_Himem"
```

Reply:
From what I learnt at IDF, it probably uses pipetask instead of payload. Something like this:
`pipetask:
    forcedPhotCoadd:
        requestMemory: 20000
        queue: "SLAC_Rubin_Extra_Himem"
`

Reply:
My mistake. Should be `pipetask` there and not `payload`.

Reply:
Maybe I can remove this part currently. For SLAC_Rubin_Extra_Himem, you don't need to specify it. With requestMemory it can be matched automatically.
Only "SLAC_Rubin_Merge" needs to be specified since it's brokeroff. One of the idea in my mind is to set merge jobs high priority. But we still need work to implement this part, so I didn't document it here.

Reply:
Ah, so it's special because it's brokeroff not because it's in the push mode, then maybe it's okay to leave it?

Reply:
I will leave it here. Zhaoyu and I have discussed about the example yaml files. In the example yaml files, 'queue: SLAC_Rubin_Merge' will not be defined by default. Jobs with "requestMemory > 18G" will be scheduled to SLAC_Rubin_Extra_Himem automatically. "SLAC_Rubin_Merge" will be used only when users update the yaml file to specify it.